tests/posix/posix-helpers.cmake

```diff
@@ -95,6 +95,12 @@ function(execute name)
 		match(${BIN_DIR}/${TRACER}.err ${SRC_DIR}/${TRACER}.err.match)
 	endif()
 
+	# pmemcheck is a special snowflake and it doesn't set exit code when it
+	# detects an error, so we have to look at its output
+	if (${TRACER} STREQUAL pmemcheck AND NOT ERR MATCHES "ERROR SUMMARY: 0")
```

Comment:
BTW, there is a test property for that kind of thing:
https://cmake.org/cmake/help/v3.0/prop_test/FAIL_REGULAR_EXPRESSION.html
Can be used as in:
https://github.com/pmem/syscall_intercept/blob/master/test/CMakeLists.txt#L321

Reply:
Indeed. I forgot about it.